src/components/FormikMultiSelect.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { InputLabel, Select, MenuItem } from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+const MenuProps = {
+  getContentAnchorEl: null, //TODO: remove this when material-ui gets updated to version 5
+  anchorOrigin: {
+    vertical: 'bottom',
+    horizontal: 'left',
+  },
+  PaperProps: {
+    style: {
+      maxHeight: 250,
+      width: 250,
+    },
+  },
+};
+
+function FormikMultiSelect(props) {
+  const { label, id, labelStyle, formik, options } = props;
+
+  return (
+    <>
```

Comment:
You could place these inside a [FormControl](https://v4.mui.com/api/form-control/) like in the [demo](https://v4.mui.com/components/selects/#multiple-select).

Reply:
I tried it, but it messes up the styling. I can try to fix this in another PR.

<img width="263" alt="Screen Shot 2021-10-01 at 10 56 25 AM" src="https://user-images.githubusercontent.com/40483569/135665869-4fa89fbf-6161-4142-910f-64de9e412848.png">